src/cf-notification/src/cf-notification.less

```diff
@@ -0,0 +1,44 @@
+/* ==========================================================================
+   Capital Framework
+   Notifications Styling
+   ========================================================================== */
+
+//
+// Theme variables
+//
+
+// Color variables
```

Comment:
Don't include it! Use our brand colors instead. These are now available to any component importing cf-core - see #561